pyqtgraph/graphicsItems/TargetItem.py

```diff
@@ -247,7 +247,7 @@ def generateShape(self):
         return dti.map(tr.map(self._path))
 
     def mouseDragEvent(self, ev):
-        if not self.movable or int(ev.button() & QtCore.Qt.LeftButton) == 0:
+        if not self.movable or int(ev.button() & QtCore.Qt.MouseButton.LeftButton) == 0:
```

Comment:
Casting to ```int``` likely doesn't work in PyQt6.

Reply:
Nice catch, fixed.